### DISCUSSION TITLE
How to show text based on a condition of two form fields

### DISCUSSION THREAD

**Discussion by spointecker** [Budibase/budibase#13010] [Help] [answered]
Hi there,

I have two form fields.
1. text field for an email domain e.g. "example.com"
2. text field for a contact email address e.g. "info@example.com"

Now I want to show an info message IF the two domain names of theses fields do not match. I tried it like this with split and last filter but it does not work:

![Bildschirmfoto 2024-02-12 um 09 39 46](https://github.com/Budibase/budibase/assets/8910787/2a7b4215-b45b-4791-ad57-baf6da2e9691)
![Bildschirmfoto 2024-02-12 um 09 39 42](https://github.com/Budibase/budibase/assets/8910787/89e9e041-43f8-43c1-b9f0-f29b240de792)

**Comment by ConorWebb96** [comment]
Hey @spointecker,

Heres how i've setup my conditions. You may need to update to the latest version of Budibase as I'm unsure what version you are on.
<img width="890" alt="Screenshot 2024-02-16 at 10 51 47" src="https://github.com/Budibase/budibase/assets/126772285/747833df-6f9d-4ddd-99f6-823db3138566">

This worked for me, I did add an additional condition to prevent it from displaying whenever domain was empty. But aside from that I didn't do anything else.

![20240216_105348](https://github.com/Budibase/budibase/assets/126772285/8055b7c4-3c86-4974-b119-06e8f3bee78c)

You can see I also checked whether your handleBars code was outputting the correct thing within the header.

Can you provide an app export so I can take a further look at this, as I can't see a reason why its working for me and not you.

**Comment by spointecker** [comment]
Hi @ConorWebb96 ,
thanks for your response. I have tried it again but it does not work. I am on the latest version, fyi.

![Bildschirmfoto 2024-02-19 um 09 44 55](https://github.com/Budibase/budibase/assets/8910787/1bd4b2f4-4185-4172-9a37-14f4c61e4373)

How should I send you the app export?

**Reply by ConorWebb96** [reply]
Hey @spointecker,

You can upload it here on this thread. Or you can send it to support@budibase.com if it contains sensitive information.

**Reply by spointecker** [reply]
Hi @ConorWebb96 ,
I just sent the export to the support email address. thanks in advance.

**Reply by ConorWebb96** [reply, marked as answer]
Hey @spointecker,

Thank you very much for your app. I've found the issue its related to the spacing within your handleBars condition binding. Can you copy and paste this exactly `{{ last (split Form 1.Fields.company_contact_email "@") }}` into the binding area highlighted in the below screenshot. I hope this fixes your issue.

<img width="891" alt="Screenshot 2024-02-19 at 13 55 34" src="https://github.com/Budibase/budibase/assets/126772285/a3367c31-2d1e-4717-a452-2813f96d724f">

**Reply by spointecker** [reply]
Thank you so much, it is working now!
